src/material-examples/button-types/button-types-example.html

```diff
@@ -72,3 +82,7 @@ <h3>Mini Fab Buttons</h3>
   </button>
   <a mat-mini-fab routerLink=".">Link</a>
 </div>
+
+<!-- Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
This copyright doesn't have to go here; it's added automatically from the docs site

Reply:
Okay thanks for the review! Copyright is removed!